docs/generating_data.md

```diff
@@ -0,0 +1,46 @@
+# Generating data
+
+## Introduction
+
+This guide explains how to populate an instance with some realistic data
+using [`GenController`](controllers.md#gencontroller).
+
+## Prerequisites
+
+- Before starting, make sure you read and understood [how to run a load-test locally](local_loadtest.md).
+
+## Configuration
+
+First of all, the type of `UserController` to be used should be set to
+`generative` as explained [here](loadtest_config.md#usercontrollerconfiguration).
+
+### Copy and modify needed configuration file
+
+```sh
+cp config/gencontroller.default.json config/gencontroller.json
```

Comment:
I think this will change to sample.json after @isacikgoz's change.

Reply:
Yes, I think we can replace everything in one go when we merge #257.